htdocs/societe/class/societe.class.php

```diff
@@ -3768,11 +3774,14 @@ function getOutstandingBills($mode='customer')
 					&& $obj->fk_statut != 2)   // Not classified as paid
 				//$sql .= " AND (fk_statut <> 3 OR close_code <> 'abandon')";		// Not abandonned for undefined reason
 				{
-					$paiement = $tmpobject->getSommePaiement();
-					$creditnotes = $tmpobject->getSumCreditNotesUsed();
-					$deposits = $tmpobject->getSumDepositsUsed();
+
 					$outstandingOpened+=$obj->total_ttc - $paiement - $creditnotes - $deposits;
 				}
+
+                if($mode == 'supplier' && $obj->type == FactureFournisseur::TYPE_CREDIT_NOTE && $obj->fk_statut == FactureFournisseur::STATUS_CLOSED && !$tmpobject->isCreditNoteUsed()) { //if credit note is converted but not used
```

Comment:
Why having this for supplier invoice and not customer invoice.
Supplier should works the same way than customer.

Reply:
This problem has been noticed on supplier side, if you say so, customer side have probably same problem